test/parallel/test-stream-duplex-from.js

```diff
@@ -401,3 +402,193 @@ function makeATestWritableStream(writeFunc) {
     assert.strictEqual(d.writable, false);
   }));
 }
+
+{
```

Comment:
Nit: I'd really prefer if we could start adding a bit more documentation to each individual test that briefly describes what is expected and what that test is verifying.

Reply:
Sorry but I don't have more time to work on this at the moment. Will definitely think about it the next time I open a PR.